doc/admin-guide/files/records.config.en.rst

```diff
@@ -1347,6 +1347,25 @@ Parent Proxy Configuration
    ``2`` Mark the host down. This is the default.
    ===== ======================================================================
 
+.. ts:cv:: CONFIG proxy.config.http.parent_proxy.enable_parent_timeout_markdowns INT 0
+   :reloadable:
+   :overridable:
+
+   Enables (``1``) or disables (``0``) parent proxy mark downs due to inactivity
+   timeouts.  The default is disabled (``0``).  This should be used for specific
+   remap entries using ``header_rewrite`` to enable inactivity timeout markdowns rather
+   than enabling this globally
+
+.. ts:cv:: CONFIG proxy.config.http.parent_proxy.disable_parent_markdowns INT 0
+   :reloadable:
+   :overridable:
+
+   Enables (``1``) or disables (``0``) parent proxy markdowns.  This is useful
+   if a parent entries in a parent.config line are VIP's and one doesn't wish
+   to mark down a VIP which may have several origin or parent proxies behind
+   this load balancer.  This should be used for specific remap entries using
+   ``header_rewrite`` rather than disabling parent mark downs globally.
```

Comment:
Would it be cleaner to just have disable_parent_markdowns:
0 - parent markdowns enabled.
1 - (default) parent markdowns disabled due to timeouts.
2 - all parent markdowns disabled.

Reply:
@ywkaras We want to do this using overridable configuration parameters.  One wouldn't override both enabling timeout markdowns and disabling markdowns at the same time.  I can add this to the documentation if it's not clear that this would not work.  I prefer it as it is.  I'll update the documentation though if you think it's necessary.